README.md

```diff
@@ -1 +1,95 @@
-# angular-scope-watch-readme
\ No newline at end of file
+# $scope.$watch and $scope.$watchCollection
+
+## Overview
+
+We can observe changes to variables using `$scope.$watch` - this is awesome, but we need to make sure we're smart with them!
+
+## Objectives
+
+- Describe watching Models with $scope.$watch
+- Describe $scope.$watchCollection
+- Describe deep Model watching
+- Write a $scope.$watch function
+- Write a $scope.$watchCollection function
+
+## Watching models
+
+In our controllers, we can watch when values update using `$scope.$watch`. We pass through two functions to this - the first one is a function that returns the value we want to watch. The second is the function that gets called when the value is updated.
+
+```js
+function SomeController($scope) {
+	var ctrl = this;
+	ctrl.search = '';
+
+	$scope.$watch(function () {
+		return ctrl.search;
+	}, function (newValue, oldValue) {
+		console.log('value updated!');
+	});
+}
+
+angular
+	.module('app')
+	.controller('SomeController', SomeController);
+```
+```html
+<input ng-model="ctrl.search" />
+```
+
+When a user types into our input, our function will get called with it's old value and it's new value. For instance, if we typed the letter `r` into our input, it would get called with `oldValue` equal to `''` and `newValue` equal to `'r'`.
+
+## Watching objects/arrays
+
+We've looked at observing strings, now let's watch objects/arrays. We can do this with ``$scope.$watchCollection`
+
+
+```js
+function SomeController($scope) {
+	var ctrl = this;
+	ctrl.collection = {
+		id: 5,
+		name: 'Test'
+	};
+
+	$scope.$watchCollection(function () {
+		return ctrl.collection;
+	}, function (newValue, oldValue) {
+		console.log('value updated!');
+	});
+}
+
+angular
+	.module('app')
+	.controller('SomeController', SomeController);
+```
+
+This will then fire out whenever we change any item in that object! However, this doesn't work that well when we change a value inside an object inside an object.
```

Comment:
Clarify this a bit - here, we're only watching when objects are added or removed from the collection, yes? It won't actually notice if something changes on the collection.